include/mbedtls/ssl_ticket.h

```diff
@@ -109,6 +109,33 @@ int mbedtls_ssl_ticket_setup( mbedtls_ssl_ticket_context *ctx,
     mbedtls_cipher_type_t cipher,
     uint32_t lifetime );
 
+/**
+ * \brief           Rotate session ticket key to new specified key.
+ *                  Provides for external control of session ticket
+ *                  encryption key rotation.
+ *
+ * \param ctx       Context to be set up
+ * \param name      Session ticket encryption key name
+ * \param nlen      Session ticket encryption key name len
```

Comment:
Style: s/len/length/ (avoid abbreviations in user documentation) and add "in bytes". (Same for klen below, especially important there are key sizes in bits are common).